gpjax/variational.py

```diff
@@ -0,0 +1,101 @@
+import abc
+from typing import Optional, Dict, Callable
+import jax.numpy as jnp
+from chex import dataclass
+
+import tensorflow_probability.substrates.jax.bijectors as tfb
+
+from .gps import Posterior
+from .types import Array, Dataset
+from .config import add_parameter, Identity, Softplus
+from .utils import I, concat_dictionaries
+
+import distrax
+
+Diagonal = distrax.Lambda(forward=lambda x: jnp.diagflat(x), inverse=lambda x: jnp.diagonal(x))
+
+FillDiagonal = distrax.Chain([Diagonal, Softplus])
+FillTriangular = distrax.Chain([tfb.FillTriangular()])
```

Comment:
Can https://github.com/deepmind/distrax/blob/master/distrax/_src/bijectors/triangular_linear.py be used here?